Source/FlowEditor/Private/Nodes/FlowNodeBlueprintFactory.cpp

```diff
@@ -34,13 +38,14 @@ class SFlowNodeBlueprintCreateDialog final : public SCompoundWidget
 {
 public:
 	SLATE_BEGIN_ARGS(SFlowNodeBlueprintCreateDialog) {}
+		SLATE_ARGUMENT(TSubclassOf<UFlowNodeBase>, ParentClass)
```

Comment:
I generalized this factory so that it could make both of the Blueprint types (Node and AddOn) with thin subclasses, below.